keyboards/darkproject/kd87b_ltd/rules.mk

```diff
@@ -0,0 +1,12 @@
+# Build Options
+#   change yes to no to disable
+#
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = spi_flash
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+# Ultrafast custom matrix scanning code
```

Comment:
```suggestion
EEPROM_DRIVER = wear_leveling
WEAR_LEVELING_DRIVER = spi_flash
```

Reply:
Can you please explain why you suggest to remove support for WFI instruction?

Reply:
It will be defined by default about a month from now (at least on master branch). See #21353.